docs/configuration/git-providers/generate-bitbucket-server-secrets.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
The instructions to generate these values were very hard.

1. You had to run 4 commands
2. You had to open files
3. You had to remove lines from files
4. You had to remove line breaks

Now you just run the script and copy the values